io.gitlab.evtest_qt.evtest_qt.json

```
@@ -1,7 +1,7 @@
 {
     "app-id": "io.gitlab.evtest_qt.evtest_qt",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.15",
+    "runtime-version": "5.15-21.08",
```

I guess it can be bumped to `5.15-22.08` by now.
```suggestion
    "runtime-version": "5.15-22.08",
```

---

https://discourse.flathub.org/t/freedesktop-sdk-22-08-0-released/2871/3

---

It should not (yet)